testcases/extc_05_fio.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Run fio controlled by environment variables
+#
+# Requires that:
+#
+# shellcheck disable=SC2119
+#
+CIJ_TEST_NAME=$(basename "${BASH_SOURCE[0]}")
+export CIJ_TEST_NAME
+# shellcheck source=modules/cijoe.sh
+source "$CIJ_ROOT/modules/cijoe.sh"
+test::enter
+
+: "${FIO_BIN:=fio}"
+
+: "${FIO_ARGS:=}"
+
+: "${FIO_NRUNS:=1}"
+: "${FIO_BS_LIST:=512}"
+: "${FIO_IODEPTH_LIST:=1}"
```

Comment:
Does this evaluate to a list? I don't think I understand the semantics of `for FIO_IODEPTH in $FIO_IODEPTH_LIST; do` in l.26

Reply:
They assign the variable when it is unset or null. Basically just providing a default-value.

Reply:
Ah ok, cool. But is the default value a list? Or am I misunderstanding the semantics of the `for xx in xx_list; do` construct? I feel like it should iterate over a list, but the default value being `1` doesn't seem listy to me

Reply:
Ah okay, I think I understand it now.. Found a bash example that described it